### PR TITLE
FABN-1543 TypeScript declaration file in fabric-common module does not match with the actual implementation

### PR DESCRIPTION
**Pull request by ntchjb**

### CODE REVIEW

**Comment by heatherlp** [comment]
hey @ntchjb Thanks for the contribution! Looks like you need to sign your commit to get the check to pass `git commit -s`

**Comment by ntchjb** [comment]
Thanks for the reply. I recently signed off every commits in this PR.

File: fabric-common/types/index.d.ts [hunk 1]
```diff
@@ -257,6 +257,11 @@ export class EventService extends ServiceAction {
 	isStarted(): boolean;
 }
 
+export interface StartEventRequest {
+	targets: Eventer[];
```
Comment (bestbeforetoday) [inline fabric-common/types/index.d.ts:261]: I think this is actually an optional property:
```
targets?: Eventer[];
```
the JSDoc would need to be updated to show it as optional too.

File: fabric-common/types/index.d.ts [hunk 2]
```diff
@@ -257,6 +257,11 @@ export class EventService extends ServiceAction {
...
+	requestTimeout: number;
```
Comment (bestbeforetoday) [inline fabric-common/types/index.d.ts:262]: I think this is also optional

**Review by bestbeforetoday — APPROVED** [review]
Spotted a couple of places where I think TypeScript definition could more closely match the implementation code but it's a good step in the right direction

**Pull request merged**